ui/src/overridableMapping.js

```diff
@@ -71,4 +72,5 @@ export const overriddenCmps = {
   "Acquisition.OrderLine": OrderDetailsLine,
   "DocumentCirculation.Extras": DocumentCirculationExtras,
   "ItemCirculation.backoffice.shelf": ItemCirculationShelf,
+  "DocumentDetails.DocumentItems": DocumentItems,
```

Comment:
```suggestion
  "DocumentDetails.DocumentItems.layout": DocumentItemsLayout,
```
Check my comment below for more details